kernel/constr.ml

```diff
@@ -99,7 +103,7 @@ type ('constr, 'types, 'sort, 'univs) kind_of_term =
   | Const     of (Constant.t * 'univs)
   | Ind       of (inductive * 'univs)
   | Construct of (constructor * 'univs)
-  | Case      of case_info * 'constr * 'constr * 'constr array
+  | Case      of case_info * 'constr * ('constr, 'univs) case_invert * 'constr * 'constr array
```

Comment:
Just wondering if this would have been a good opportunity to use an inline record.

Reply:
I felt like it would be too painful compared to adding a `_, ` in most places.